### ISSUE TITLE
Cfitsio.py download link not working

### ISSUE THREAD

**Issue by GabrielMissael** [brinckmann/montepython_public#278]
Hello!

I'm trying to install the WMAP wrapper, and when running `./waf configure --install_all_deps`, it appears the following error message:

```
Checking for library ['cfitsio']                                           : not found
cfitsio not found
check that cfitsio_prefix or cfitsio_lib and cfitsio_include command line options point toward your cfitsio install
or check that cfitsio is compiled in 64 bit
cfitsio not found, try to install it
Install 'cfitsio3280.tar.gz'
download from ftp://heasarc.gsfc.nasa.gov/software/fitsio/c/cfitsio3280.tar.gz
curl ftp://heasarc.gsfc.nasa.gov/software/fitsio/c/cfitsio3280.tar.gz build/cfitsio3280.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (67) Access denied: 550
```

Looking in the web, I found that this link actually works and corresponds to the same cfitsio version: [https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio3280.tar.gz](https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio3280.tar.gz)

**Comment by brinckmann** [comment]
Thanks for pointing out they changed the link and for providing the new one!

Best,
Thejs

**Comment by dchooper** [comment]
I have implemented this bugfix, it will be included in MP v3.6.0 (coming soon!). Thanks for pointing it out!
